llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -1,5 +1,7 @@
 //===- ControlPoint.cpp - Synthesise the yk control point -----------------===//
 //
+// FIXME: update this comment.
```

Comment:
Is there a reason we're not updating it in this PR?

Reply:
Only that I've been focusing on the runtime side.

I'll need to get this one up to scratch before we can merge https://github.com/ykjit/ykllvm/pull/17

Reply:
This is #17?

Reply:
sorry, i meant, before we can merge https://github.com/ykjit/yk/pull/454

Reply:
I'm also unclear: are you going to fix the comment or ... ?

Reply:
Yes, I'll fix it tomorrow morning :)